R/texreg.R

```diff
@@ -1925,6 +1925,467 @@ print.texregTable <- function(x, ...) {
+    } else {
+        stop("'signif.outer' does not correspond to the intervals provided.")
+    }
+    
+    
+    dataframe <- cbind(dataframe, signif)
+    
+    if (length(co) == 0) {
+        stop(paste("No coefficients available. Was the 'omit.coef' argument", 
+                   "misspecified? If coefficients were renamed using the",
+                   "'custom.coef.names' argument, 'omit.coef' refers to the renamed",
+                   "coefficients."))
+    }
+    
+    # omit.coef
+    if (!is.na(omit.coef)) {
+        dataframe <- dataframe[!grepl(omit.coef, dataframe$co.names), ]
+    }
+    
+    # custom coef. name
+    if (length(custom.coef.names) > 0) {
+        levels(dataframe$co.names) <- custom.coef.names
+    }
+    
+    # custom model name
+    if (length(custom.model.names) > 0) {
+        levels(dataframe$lab) <- custom.model.names
+    }
+    
+    # ggplot functions   
+    if (type == "facet") {
+        if (length(reorder.coef) > 0) {
+            # reorder levels 
+            dataframe$co.names <- ordered(dataframe$co.names, levels = (unique(as.character(dataframe$co.names))))
+            # reorder coef
+            reorder.coef<- rev(reorder.coef)
+            dataframe$co.names <- factor(dataframe$co.names, levels(dataframe$co.names)[reorder.coef])
+        } else {
+            # reorder levels 
+            dataframe$co.names <- ordered(dataframe$co.names, levels = (unique(as.character(dataframe$co.names))))
+            # reorder coef in original order
+            startlevel<- c(length(dataframe$co.names):1)
+            dataframe$co.names <- factor(dataframe$co.names, levels(dataframe$co.names)[startlevel])
+        }
+        
+        if (length(custom.coef.map) > 0) {
+            # keep only coefficients selected by the user and replace NAs if any
+            idx <- is.na(custom.coef.map)
+            custom.coef.map[idx] <- names(custom.coef.map)[idx]
+            selectcoef <- names(custom.coef.map)
+            keepcoef <- paste(selectcoef, collapse = "|")
+            dataframe <- dataframe[grepl(keepcoef, dataframe$co.names), ]
+            # resize number of levels in factor to avoid error
+            dataframe$co.names <- ordered(dataframe$co.names, levels = (unique(as.character(dataframe$co.names))))
+            # rename selected coefficients
+            renamedcoef <- paste(unlist(custom.coef.map), sep = ", ")
+            levels(dataframe$co.names) <- renamedcoef
+            # invert order factors for plot
+            coeforder<- c(length(dataframe$co.names):1)
+            dataframe$co.names <- factor(dataframe$co.names, levels(dataframe$co.names)[coeforder])
+        }
+        
+        p <- ggplot(dataframe, aes(co.names, co)) + 
+            geom_hline(yintercept = 0, 
+                       lty = 2, lwd = 1, 
+                       colour = "grey50") +
+            geom_errorbar(aes(ymin = lower.outer, ymax = upper.outer), 
+                          lwd = 1, 
```

Comment:
These arguments should be indented under `ymin`, not `aes`. Have you tried to highlight the whole function code and press Ctrl-I in RStudio to automatically fix the indentation for the whole function? I think that would be helpful.

Reply:
I did it several times and that's how it makes it. Now I changed it the way you are suggesting

Reply:
Weird. But okay...